src/bootupd.rs

```diff
@@ -586,7 +571,7 @@ pub(crate) fn client_run_migrate_static_grub_config() -> Result<()> {
             "ostree repo 'sysroot.bootloader' config option is currently set to: '{}'",
             bootloader
         );
-    }
+    };
```

Comment:
Hmm, that semicolon shouldn't be necessary, but I'm surprised clippy doesn't complain about it.